onnxscript/function_libs/torch_lib/tensor_typing.py

```diff
@@ -56,6 +56,7 @@
 ]
 
 TTensor = TypeVar("TTensor", bound=_TensorType)
+TTensor2 = TypeVar("TTensor2", bound=_TensorType)
```

Comment:
What kind of problem will be caused if we don't have such duplicates? If there is a reason, it'd better to leave here as comments for a reminder.

Reply:
Done. Added a comment